snuba/datasets/outcomes_processor.py

```diff
@@ -33,7 +26,13 @@ def process_message(self, value, metadata) -> Optional[ProcessedMessage]:
             "timestamp": _ensure_valid_date(
                 datetime.strptime(value["timestamp"], settings.PAYLOAD_DATETIME_FORMAT),
             ),
-            "outcome": value["outcome"],
+            "category": value.get(
+                "category", DataCategory.ERROR
+            ).value,  # if category is None, default to error for now TODO: change security to error?
+            "size": value.get(
+                "size", 1
+            ),  # should probably be a default value of 1 for size and none
+            "outcome": value.get("outcome", None),
```

Comment:
You will not be able to make the change to the schema and to the processor at the same time, otherwise this will fail as soon it is deployed as the DB schema would not have been updated yet.